### PR TITLE
Addition of account_recovery_setting recovery_mechanisms

### PR DESCRIPTION
**Pull request by mariux**
- Addition of account_recovery_setting recovery_mechanisms
- PR improvements
- PR changes
- PR changes
- release: Prepare v0.5.0 release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,9 @@ A [Terraform] module for deploying and managing
 on [Amazon Web Services (AWS)][AWS].
 
 *This module supports Terraform v0.14, v0.13 as well as v0.12.20 and above
-and is compatible with the terraform AWS provider v3 as well as v2.54 and above.*
+and is compatible with the terraform AWS provider v3.19 and above.*
```
Comment (soerenmartius) [inline README.md:16]:
```suggestion
and is compatible with the Terraform AWS provider v3.19 and above.*
```

File: README.md [hunk 2]
```diff
@@ -13,7 +13,9 @@ A [Terraform] module for deploying and managing
...
+
+The last version supporting terraform AWS provider v2.x is v0.4.1.
```
Comment (soerenmartius) [inline README.md:18]:
```suggestion
The last version supporting Terraform AWS provider v2.x is v0.4.1.
```

**Review by soerenmartius — DISMISSED** [review]
lgtm! Please see the minor suggestions.

**Comment by mariux** [comment]
CI updated @soerenmartius

**Review by soerenmartius — APPROVED** [review]
lgtm!

**Comment by svenlito** [comment]
@mariux can you publish a release for this?

**Comment by mariux** [comment]
@svenlito just released 0.5.0 ... thank you ;) let us know if something is not working as expected by opening a new issue ;) enjoy

**Pull request merged**